tests/integ/test_tf_cifar.py

```diff
@@ -19,12 +22,34 @@
 from tests.integ import DATA_DIR, REGION
 from tests.integ.timeout import timeout_and_delete_endpoint, timeout
 
+PICKLE_CONTENT_TYPE = "application/python-pickle"
```

Comment:
use single quotes instead of double quotes for consistency with the rest of the SDK